### DISCUSSION TITLE
Optimization suggestions for some programs that need to lock the cursor

### DISCUSSION THREAD

**Discussion by Qs315490** [rustdesk/rustdesk#3827] [General]
部分应用使用锁定鼠标的方式来获取鼠标移动轨迹, 内部独立出一套指针系统. 现有的远程控制都是采用修改光标位置的方式来实现控制光标. 这种方式会导致该应用内的光标异常, 要像摇杆一样操作鼠标才能勉强正常使用. 从安卓控制电脑我发现本地光标位置会反馈到手机上, 只要能把程序修改的光标位置发到控制端或修改控制逻辑就可以解决问题.
云游戏倒是能解决我的需求, 可惜我这边使用的是隔绝外网的内部网络.
这问题常常出现在游戏上.

**Comment by rustdesk** [comment]
The "issue" is not only for me, I hope more guys can see it and contribute. So English Please.

**Comment by Qs315490** [comment]
I'm sorry, my English is not good. Can't translate these
